packages/chip/src/lib/Chip.stories.tsx

```diff
@@ -44,24 +45,6 @@ export const _Avatar: StoryObj<typeof Chip> = {
   render: (args: ChipProps) => <Chip {...args} avatar={<Avatar>A</Avatar>} />,
 };
 
-/** Status styling is achieved by using size `small` */
-export const _Status: StoryObj<typeof Chip> = {
-  render: () => (
-    <>
-      Item 1 <Chip color="success" label="Approved" size="small" />
-      <br />
-      Item 2 <Chip color="info" label="Info" size="small" />
-      <br />
-      Item 3 <Chip color="warning" label="Pending" size="small" />
-      <br />
-      Item 4 <Chip color="error" label="Declined" size="small" />
-      <br />
-      Item 5 <Chip color="secondary" label="Neutral" size="small" />
-      <br />
-    </>
-  ),
-};
-
 /** Inline counter styling is achieved by using size `small`. For counters that affix to the corner of an element see `Badge`. */
 export const _InlineCounter: StoryObj<typeof Chip> = {
```

Comment:
This would use the statuschip now. I don't remember if ux has this  spec'd out anymore, but if they don't we could remove this story altogether vs just moving it to statuschip. Maybe we should update the statuschip description to include this use.

Reply:
I feel like if we're going to separate StatusChip, we should make a separate InlineCounter component.

Reply:
Sounds like InlineCounter can be removed.